.env.example

```diff
@@ -58,7 +61,7 @@ PGSSLMODE=
 # @see: https://github.com/calendso/calendso/issues/263
 # @see: https://next-auth.js.org/configuration/options#nextauth_url
 # Required for Vercel hosting - set NEXTAUTH_URL to equal your NEXT_PUBLIC_WEBAPP_URL
-NEXTAUTH_URL='http://localhost:3000'
+NEXTAUTH_URL='http://app.cal.local:3000'
```

Comment:
Same here I guess?

Reply:
Reverting back to localhost as well, forgot to do that when introduced the new orgs env var. Is there a reason we have this splitted in so many env vars? could we merge them into one?